content/syllabus-written-assignments/file-directory-naming/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+content_type: project
+flavours:
+  - java
+  - javascript
+  - markdown
+  - python
+prerequisites:
+  hard:
+    - topics/file-and-directory-naming/general
+protect_main_branch: false
+ready: true
+submission_type: repo
+title: File and Directory Naming
+---
+
+Please answer the following questions in markdown files and submit following the instructions below.
+
+## Questions
+
+1. Here are some directory names that follow different naming conventions: Foo_Bar, foo_bar, FooBar, main, fooBar. Which of the directory names follows the convention used in your language?
```

Comment:
why is main in there? It'll be much easier to auto mark if there is only one correct answer.